src/applications/mhv-medications/containers/RefillPrescriptions.jsx

```diff
@@ -117,6 +118,7 @@ const RefillPrescriptions = ({ isLoadingList = true }) => {
   };
 
   useEffect(() => {
+    dispatch(clearFillNotification);
```

Comment:
probably not much of a performance increase but you could add a check to clear only if there's an existing notification

Reply:
TY. Done!